perfomance_tests/filter/filter_plain_metrics_test.go

```diff
@@ -0,0 +1,83 @@
+package filter
+
+import (
+	"bufio"
+	"io"
+	"math/rand"
+	"os"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/moira-alert/moira/filter"
+)
+
+func BenchmarkProcessIncomingMetric(b *testing.B) {
+	plainPatterns, err := loadPlainPatterns()
+	if err != nil {
+		b.Errorf(err.Error())
+	}
+
+	patternsStorage, err := createPatternsStorage(plainPatterns, b)
+	if err != nil {
+		b.Errorf("Can not create new patterns storage %s", err)
+	}
+
+	testMetricsLines := generateMetrics(patternsStorage, b.N)
+
+	runBenchmark(b, patternsStorage, testMetricsLines)
+}
+
+func loadPlainPatterns() ([]string, error) {
```

Comment:
loadPlainPatterns() and loadTaggedPatterns() are very close. They should be replaced to one method loadPatterns(filename).

Reply:
Good point, updated!